$OpenBSD: patch-configure,v 1.11 2010/09/30 07:26:05 ajacoutot Exp $

XXX make it compile with gtk+2 2.22 (*_DISABLE_DEPRECATED)
lqt_gtk.c:944: warning: implicit declaration of function 'GTK_OBJECT_FLAGS'
lqt_gtk.c:944: error: lvalue required as left operand of assignment

--- configure.orig	Tue Feb 23 21:43:36 2010
+++ configure	Wed Sep 29 19:50:57 2010
@@ -14553,7 +14553,7 @@ fi
     VORBIS_LIBS="-L$prefix/lib"
   fi
 
-  VORBIS_LIBS="$VORBIS_LIBS -lvorbis -lm"
+  VORBIS_LIBS="$VORBIS_LIBS -lvorbis -logg -lm"
   VORBISFILE_LIBS="-lvorbisfile"
   VORBISENC_LIBS="-lvorbisenc"
 
@@ -14713,11 +14713,11 @@ if test "x$with_lame" != "xno"; then
 OLD_CFLAGS=$CFLAGS
 OLD_LIBS=$LIBS
 
-if test x$have_vorbis = xtrue; then
-LIBS="$LIBS -lmp3lame -lvorbis -lm"
-else
+#if test x$have_vorbis = xtrue; then
+#LIBS="$LIBS -lmp3lame -lvorbis -lm"
+#else
 LIBS="$LIBS -lmp3lame -lm"
-fi
+#fi
 
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for lame" >&5
@@ -15837,7 +15837,7 @@ fi
 
 
 
-GTK_CFLAGS="$GTK_CFLAGS -DGTK_DISABLE_DEPRECATED"
+GTK_CFLAGS="$GTK_CFLAGS"
 
 
 have_libdv="false"
